avm/ptn/policy-insights/remediation/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,152 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
```

Comment:
Please update metadata name and description to max test related values (you can reference any res module)

Reply:
Same here, please update the description as well